sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/api/Azure.Monitor.OpenTelemetry.Exporter.net6.0.cs

```diff
@@ -1,3 +1,15 @@
+namespace Azure.Monitor.OpenTelemetry.CustomEvents
```

Comment:
This namespace doesn't match the one used in the classes.
Is this an incomplete refactor?

Reply:
yes, fixed.